zrml/global-disputes/src/lib.rs

```diff
@@ -221,6 +225,9 @@ mod pallet {
         ) -> DispatchResultWithPostInfo {
             let owner = ensure_signed(origin)?;
 
+            let market = T::MarketCommons::market(&market_id)?;
```

Comment:
Because of Malte's suggestion in the ZIP-0 PR I am in favour of not describing every possible error in the doc comments. I would leave just pretty general errors like they are described now. But I see your point.